src/Symfony/Component/Security/Core/Exception/AccessDeniedException.php

```diff
@@ -31,9 +33,9 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public function setAttributes(array|string $attributes)
+    public function setAttributes(array|string|Expression $attributes)
```

Comment:
BC: This change breaks code that extends `AccessDeniedException` and overrides this method.

Reply:
As mentioned in #46986 this seems like a bug to me since it was always possible for the value to be an instance of `Expression`. If it's decided to treat this as a bugfix, I don't think the change will be a problem since it's probably not a common use case to extend this exception and override that particular method.
On the other, if it's not treated as a bug, a new method could be.

Reply:
I checked out the code locally and reverted this change: tests still pass and the Expression is auto-cast to string. No need for this change to me.

Reply:
Yes, the problem is with low-deps, bumping the min version would then be required.

Reply:
then let's bump, it's a far better move compared to a BC break :)

Reply:
Done